### DISCUSSION TITLE
Status of prometheus exporter

### DISCUSSION THREAD

**Discussion by bjor-joh** [open-telemetry/opentelemetry-java#5266] [Q&A] [answered]
Hi

I can see that the prometheus exporter is currently in alpha. 
Can you share anything regarding what the plans are for this exporter?
When can beta and/or stable be expected?

Thanks
Björn

**Comment by jack-berg** [comment, marked as answer]
There is still a fair amount that needs to be done before we can stabilize the prometheus exporter:
- The prometheus exporter in the spec is [experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/prometheus.md), and would need to be marked stable before we could follow suite.
- The prometheus exporter specification is mostly just waiting on the [prometheus and openmetrics compatibility](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/prometheus_and_openmetrics.md) document, which dictates how we translate from the opentelemetry data model to prometheus / open metrics. 
- There a number of things that still need to be implemented in the prometheus exporter to be spec compliant, including #4390, #5141, and likely some additional things that are yet filed as issues.

We're happy to accept contributions if you're interested in helping to accelerate this effort 🙂

**Reply by bjor-joh** [reply]
Thank you 🙏🏻

**Reply by bikehead** [reply]
I would love to experiment and contribute to this exporter.  Are there any instructions on how to set it up for use?
